packages/adp-tooling/templates/package.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "<%= package.name %>",
+    "version": "0.1.0",
+    "private": true,
+    "description": "<%= package.description %>",
+    "keywords": [
+        "ui5",
+        "openui5",
+        "sapui5"
+    ],
+    "dependencies": {},
+    "devDependencies": {
+        "@sap-ux/backend-proxy-middleware": "^0.6.61",
+        "@sap-ux/preview-middleware": "^0.2.0",
+        "@sap-ux/ui5-proxy-middleware": "^1.1.31",
+        "@ui5/cli": "^3.3.2"
+    },
+    "scripts": {
+        "build": "ui5 build --clean-dest",
+        "start": "ui5 serve --open /test/flp.html#app-preview",
+        "editor": "ui5 serve --open /test/flp.html?fiori-tools-rta-mode=true#app-preview"
```

Comment:
I believe that this needs to be `fiori-tools-rta-mode=forAdaptation`, otherwise one can only do Variant Management

Reply:
Done with https://github.com/SAP/open-ux-tools/pull/1105/commits/c82e19158b8f6a7d9d6e73f9cc315b326e31866a

Reply:
i think you need to update the test snapshots as well.

Reply:
Done with https://github.com/SAP/open-ux-tools/pull/1105/commits/499655d237eac09bdb7a09b6f87d50ca00c5a357, thanks